### DISCUSSION TITLE
Is this repository still actively maintained?

### DISCUSSION THREAD

**Discussion by stan-at-work** [doonfrs/pluto_grid_plus#51] [Q&A]
Hi there,
It seems there are several PR/issues (almost all?) not answered/managed.
So just wondering if this repository is actively maintained?

**Comment by doonfrs** [comment]
Hi, I always publish the latest version to the pub.dev from time to time, it is cumulative work, I can not personally answer all issues, so it is still maintained but more contribution is needed. 🙏🙏

**Reply by VittorioParagallo** [reply]
publish a call for reviewers here and also on the original project.

**Reply by doonfrs** [reply]
https://github.com/doonfrs/pluto_grid_plus/discussions/56
